estate/models/estate_property.py

```diff
@@ -0,0 +1,57 @@
+from odoo import fields, models, api
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real estate Property Field"
+
+    name = fields.Char(required=True, string='Title')
+    description = fields.Text(string='Description')
+    postcode = fields.Char(string='Postcode')
+    date_availability = fields.Date(copy=False, default=fields.Date.add(fields.Date.today(), days=90), string='Available From')
```

Comment:
Not all months consist of 30 days, so to be more precise you can use months option.
```suggestion
    date_availability = fields.Date(copy=False, default=fields.Date.add(fields.Date.today(), months=3), string='Available From')
```